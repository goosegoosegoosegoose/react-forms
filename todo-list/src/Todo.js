import React from "react";

const Todo = ({todo, remove}) => {
    console.log(todo);
    return (
        <div className="todo">
            <p>
                {todo} <button onClick={remove}>X</button>
            </p>
            
        </div>
    )
}

export default Todo;