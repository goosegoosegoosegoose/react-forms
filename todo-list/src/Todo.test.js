import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo"

it("smoke test", () => {
    render(<Todo />);
});

it("snapshot test", () => {
    const { asFragment } = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
})