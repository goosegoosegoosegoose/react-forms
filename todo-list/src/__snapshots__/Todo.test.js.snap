// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <div
    class="todo"
  >
    <p>
       
      <button>
        X
      </button>
    </p>
  </div>
</DocumentFragment>
`;
